scripts/set_processor.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+USAGE="usage: ./set-processor [gpu|cpu]"
+
+if [ $# != 1 ]; then
+    echo "$USAGE"
+    exit 1
+fi
+
+PROCESSOR=$1
+if [ "$PROCESSOR" == "gpu" ]; then
```

Comment:
Yeah, I wondered if there was a better way, then realized this was how we already had it in the dockerfile... As ugly as it looks, I can't think of anything better.